local-proxy/src/server.ts

```diff
@@ -192,7 +195,20 @@ router.post('/invoke', async (ctx) => {
   }
 });
 
-const db = new DBHandler(resolvePath(tmpDir, 'db'));
+const dbPath = resolvePath(tmpDir, 'db');
+const initDataPath = resolvePath(rootPath, 'template_init_data.json');
+let initData: any;
+if (!existsSync(dbPath) && existsSync(initDataPath)) {
+  try {
+    initData = JSON.parse(readFileSync(initDataPath, 'utf8'));
```

Comment:
not for this PR: consider extracting the processing logic (schema validation iirc) to shared place?